src/Microsoft.OpenApi.Readers/ReferenceServices/IOpenApiVersionService.cs

```diff
@@ -3,17 +3,20 @@
 
 using Microsoft.OpenApi.Interfaces;
 using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers.ParseNodes;
+using System;
```

Comment:
Sort

Reply:
Aren't they in alphabetical order already?

Reply:
No. System should be at top.




---
In reply to: [157916787](https://github.com/Microsoft/OpenAPI.NET/pull/161#discussion_r157916787) [](ancestors = 157916787)

Reply:
This is a weird thing that happens to Visual Studio 2015 onwards. The default becomes strict alphabetical, while it used to be System and then alphabetical before 2015. The guideline we are loosely adhering to suggests we go with System, then alphabetical.

---
In reply to: [157918003](https://github.com/Microsoft/OpenAPI.NET/pull/161#discussion_r157918003) [](ancestors = 157918003,157916787)

Reply:
LOL. Ok.

---
In reply to: [157918851](https://github.com/Microsoft/OpenAPI.NET/pull/161#discussion_r157918851) [](ancestors = 157918851,157918003,157916787)